components/calendar/README.md

```diff
@@ -20,6 +20,14 @@ The `d2l-calendar` component can be used to display a responsively sized calenda
 - `selected-value` (String): Currently selected date. This should be in ISO 8601 calendar date format (`YYYY-MM-DD`) and should be localized to the user's timezone (if applicable).
 - `summary` (String): Summary of the calendar for accessibility.
 
+**Accessibility:**
```

Comment:
Do we want to switch the rest of the API doc to use tables as well?

Reply:
I would be okay with that (and can make that change over the next bit) if others are interested.